xchange-bitstamp/src/main/java/org/knowm/xchange/bitstamp/service/BitstampTradeService.java

```diff
@@ -146,14 +155,22 @@ public UserTrades getTradeHistory(TradeHistoryParams params) throws IOException
               .map(Long::parseLong)
               .orElse(null);
     }
-    BitstampUserTransaction[] txs =
-        getBitstampUserTransactions(
-            limit,
-            currencyPair,
-            offset,
-            sort == null ? null : sort.toString(),
-            sinceTimestamp,
-            sinceId);
+    BitstampUserTransaction[] txs;
+    if (currencyPair == null) {
+      txs =
+          getBitstampUserTransactions(
+              limit, offset, sort == null ? null : sort.toString(), sinceTimestamp, sinceId);
```

Comment:
just one small remark

prepare a `String sortStr = sort == null ? null : sort.toString();` before the if

Reply:
Sure, done.

Reply:
thanks